_posts/2015-09-18-zsnm3456.md

```diff
@@ -10,23 +10,23 @@ Naposledy jsem poctivě psal domácí úkoly pro StuNoMe před rokem a půl. Res
 
-Typickým případem světatvorby – či spíše reflexe různých verzí reality – jsou různé úrovně abstrakce. Jeden objekt můžeme popsat v rámci různých sytémů které budují vlastní vidění světa na základě určitých axiomů. Potom například jablko může být z botanického pohledu plod krytosemenné rostliny, z chemického pohledu shluk molekul či složitějších vazeb aminokyselin, z fyzikálního pohledu skupenství atomů a z nutričního pohledu je to zdroj vlákniny a vitaminu C…
+Typickým případem světatvorby – či spíše reflexe různých verzí reality – jsou různé úrovně abstrakce. Jeden objekt můžeme popsat v rámci různých sytémů, které budují vlastní vidění světa na základě určitých axiomů. Potom například jablko může být z botanického pohledu plod krytosemenné rostliny, z chemického pohledu shluk molekul či složitějších vazeb aminokyselin, z fyzikálního pohledu skupenství atomů a z nutričního pohledu je to zdroj vlákniny a vitaminu C…
 
-Co zde provádíme je zvýrazňování a potlačování určitých aspektů v té které verzi reality. Goodman konkrétně jmenuje několik způsobů světatvorby: _kompozice / dekompozice_ – stejnou věc můžeme rozložit – či složit více způsoby; _zvýrazňování_ – záměrně vyzdvihujeme určité aspekty světa; _uspořádání_ – jaký je vnitřní řád světa[^desitkova]; _vylučování a doplňování_ – přidává či ubírá některé prvky ze světa; _deformace_ – zkreslení či naopak korekce některých aspektů.
+Co zde provádíme je zvýrazňování a potlačování určitých aspektů v té které verzi reality. Goodman konkrétně jmenuje několik způsobů světatvorby: _kompozice / dekompozice_ – stejnou věc můžeme rozložit či složit více způsoby; _zvýrazňování_ – záměrně vyzdvihujeme určité aspekty světa; _uspořádání_ – jaký je vnitřní řád světa[^desitkova]; _vylučování a doplňování_ – přidává či ubírá některé prvky ze světa; _deformace_ – zkreslení či naopak korekce některých aspektů.
 
 [^desitkova]: Krásným příkladem je dominance desítkové soustavy v našem světě; proč slavíme „kulatá“ výročí a měříme dekády, staletí či milénia? Nejspíše proto, že máme deset prstů…
 
-Krásné na Goodmanově [irrealismu](https://en.wikipedia.org/wiki/Irrealism_%28philosophy%29) je, že je prakticky neprůstřelný. I pokud jej zavrhnete, děje se tak _ve vaší verzi světa_, s čímž ovšem Goodmanova teorie tak nějak počítá.
+Krásné na Goodmanově [irrealismu](https://en.wikipedia.org/wiki/Irrealism_%28philosophy%29) je, že je prakticky neprůstřelný (odborněji nefalzifikovatelný). I pokud jej zavrhnete, děje se tak _ve vaší verzi světa_, s čímž ovšem Goodmanova teorie tak nějak počítá.
```

Comment:
Nefalzifikovatelný? To je dost divné použití v tomhle kontextu a není to pravda. Není to odborný text, je to blogpost.